cloud/spawn.go

```diff
@@ -230,6 +230,28 @@ func TerminateSpawnHost(ctx context.Context, host *host.Host, settings *evergree
 	return nil
 }
 
+func StopSpawnHost(ctx context.Context, host *host.Host, settings *evergreen.Settings, user string) error {
+	cloudHost, err := GetCloudHost(ctx, host, settings)
+	if err != nil {
+		return err
```

Comment:
Are we sure these errors are verbose enough to not be wrapped? If so that's fine but